test/remote-syslog.sh

```diff
@@ -284,10 +284,16 @@ artifact_log $title $mypod
 
 mymessage="rsyslogTestMessage-"$( date +%Y%m%d-%H%M%S )
 logger -i -p local6.info -t rsyslogTestTag $mymessage
-os::cmd::try_until_text "sudo grep $mymessage /var/log/messages | awk -F':' '{print \$4}'" " $mymessage" $MUX_WAIT_TIME
+os::cmd::try_until_text "sudo grep $mymessage /var/log/messages" ".*${mymessage}.*" $MUX_WAIT_TIME
```

Comment:
it would be better to use `journalctl` - but it would be very expensive to call `journalctl | grep something` every 0.2 seconds, which is what `try_until_text` does.   Maybe something like this?
```bash
tmpjournal=$( mktemp )
journalctl -f > $tmpjournal & jcpid=$!
logger -i -p local6.info -t rsyslogTestTag $mymessage
os::cmd::try_until_text "sudo grep $mymessage $tmpjournal" ".*${mymessage}.*" $MUX_WAIT_TIME
kill $jcpid # also need to do this in cleanup()
```
If you know that the message is going to be logged to a specific unit (e.g. rsyslog) then you can use `journalctl -f -u rsyslog`

Reply:
In this test, i've restarted rsyslog with tcp input:
https://github.com/nhosoi/origin-aggregated-logging/blob/bz1515715/test/remote-syslog.sh#L228-L235
and let fluentd output-remote-syslog.conf point it.  That is, I'm expecting to see the forwarded message is stored in /var/log/messages by rsyslogd (instead of capturing the message from journald as in this flow: logger -> journald -> fluentd -> rsyslogd).  Do you think the test case is not written as I described?

Reply:
Ok, I see.  Then it is ok to use /var/log/messages